docs/integrations/microsoft/sso-saml.mdx

```diff
@@ -0,0 +1,92 @@
+---
+description: Use SAML with Microsoft Entra ID to provide SSO to the ngrok Dashboard
```

Comment:
```suggestion
description: Use SAML with Microsoft Entra Verified ID to provide SSO to the ngrok Dashboard
```

Reply:
The Product itself is Microsoft Entra ID.  Verified ID is included in Entra ID.  In Azure the service is referred to as Entra ID.

I also believe that Verified ID is used for JWT.

Verified ID is currently included with any Microsoft Entra ID subscription, including Microsoft Entra ID Free, at no additional cost.  https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-verified-id#tabx67298482964d423da017e0eae7ba14b3